scaler/handlers.go

```diff
@@ -151,15 +160,16 @@ func (e *impl) GetMetricSpec(
 			)
 			return nil, err
 		}
+		host = "http-requests"
 		targetPendingRequests = int64(target.TargetPendingRequests)
 	}
+
 	metricSpecs := []*externalscaler.MetricSpec{
 		{
 			MetricName: host,
```

Comment:
```suggestion
			MetricName: httpRequests,
```

Reply:
Don't we want to keep interceptor as metricName when the host is `interceptor`?

Reply:
you are right